addons/web/static/src/js/fields/abstract_field_owl.js

```diff
@@ -0,0 +1,462 @@
+odoo.define('web.AbstractFieldOwl', function (require) {
+    "use strict";
+
```

Comment:
keep explanation text of AbstractField (+ add a sentence about the fact that this one is Owl)